myuw_vue/components/home/applicant/seattle.vue

```diff
@@ -23,26 +23,22 @@
             <a v-out="'Contact Registrar Office'" href="mailto:regoff@uw.edu">regoff@uw.edu</a>
           </div>
         </div>
-        <div class="row">
-          <div class="col">Phone</div>
-          <div class="col">(206) 543-4000</div>
-        </div>
         <div class="row">
           <div class="col">Location</div>
-          <div class="col">225 Schmitz Hall</div>
+          <div class="col">2nd Floor Schmitz Hall</div>
         </div>
       </div>
 
       <h3 class="h6 myuw-font-encode-sans">Registration for returning student</h3>
       <p>
```

Comment:
Hard to remember, but most text in MyUW needs class="myuw-text-md" to display at the standard size...

Reply:
thanks for the catch! made the edit :)